### ISSUE TITLE
Overlay stays visible if show: { maxZoom: 1 } only

### ISSUE THREAD

**Issue by philippfromme** [bpmn-io/diagram-js#235]
When adding overlay with configuration

```javascript
overlays.add(task, {
  html: 'FOO',
  position: {
    top: 0,
    left: 0
  },
  show: {
    maxZoom: 1
  }
});
```

Overlay stays visible (only with this exact configuration).

![foobar](https://user-images.githubusercontent.com/7633572/32061275-7a301260-ba71-11e7-9ef9-8b9b0f4ad38e.gif)

**Comment by nikku** [comment]
Fix can be reviewed. Simple typo :cry:.

**Comment by nikku** [comment]
Should I cut a patch releases for our libraries?

**Comment by nikku** [comment]
Released as `diagram-js@v0.22.1`.
